addons/website_sale/views/product_views.xml

```diff
@@ -158,9 +158,9 @@
         <field name="inherit_id" ref="product.product_template_form_view"/>
         <field name="arch" type="xml">
             <!-- add state field in header -->
-            <div name="button_box" position="inside">
+            <span id="button_website" position="replace">
```

Comment:
Would be nice to use views sequence to order the buttons instead of this hook.
Also, instead of replacing it, maybe place the button before (or after), so that if another view uses that hook, it won't break.